packages/shopify-app-remix/src/server/auth/helpers/redirect-with-exitiframe.ts

```diff
@@ -5,6 +5,7 @@ import type {BasicParams} from '../../types';
 export function redirectWithExitIframe(
   params: BasicParams,
   request: Request,
+  redirectPath: string,
```

Comment:
Why this addition? From what I can see we always pass `config.auth.path` to this argument, which is what the function body used to hard code anyway?